packages/verification-vue-component/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@certifaction/verification-vue-component",
-    "version": "3.3.16",
+    "version": "3.3.17",
```

Comment:
Why this change? This should only be changed when a new release is published with lerna.